react-basic-practice/src/components/TableFinance/index.tsx

```diff
@@ -17,9 +21,16 @@ const TableFinance: React.FC<ITablePrice> = ({ value, isBalance }) => {
       <Text fontSize="md" lineHeight="short" color={color}>
         {value}
       </Text>
-      <Text fontSize="sm" lineHeight="shorter" color="paleSky" letterSpacing="">
-        CAD
-      </Text>
+      {showCurrency && (
+        <Text
+          fontSize="sm"
+          lineHeight="shorter"
+          color="paleSky"
+          letterSpacing=""
```

Comment:
pls remove redundant code. It's empty value

Reply:
i done